### PR TITLE
Add script to test all models in op

### PR DESCRIPTION
**Pull request by NicoYuan1986**
Signed-off-by: nico <cheng.yuan@zilliz.com>

### CODE REVIEW

File: tests/testcases/hub/execute_hub_operators.py [hunk 1]
```diff
@@ -152,31 +192,42 @@ def execute_ops(op_clone_list):
...
         # execute python code from op readme
-        logger.debug("Running extracted readme for %s" % op_single_path)
-        test_case_name_backup = test_case_path + "/test_" + op_name_single + ".py"
-        res = os.system(f"python3 {test_case_name_backup} >> {log_path}")
-        if 0 != res:
-            logger.error("Fail to run readme for %d: operator %s" % (op_id, op_address))
+        is_succeed = True
+        for model in all_models:
+            model_name = model.rsplit('\'', 2)[1]
+            modify_specific_model(test_case_name, model_name)
```
Comment (binbinlv) [inline tests/testcases/hub/execute_hub_operators.py:208]: I think here should be modify_specific_model(test_case_name_backup, model_name). And move this line after "test_case_name_backup = test_case_path + "/test_" + op_name_single + ".py""

File: tests/testcases/hub/execute_hub_operators.py [hunk 2]
```diff
@@ -152,31 +192,42 @@ def execute_ops(op_clone_list):
...
+            logger.debug("Running extracted readme for %s - %s" % (op_single_path, model_name))
+            test_case_name_backup = test_case_path + "/test_" + op_name_single + ".py"
+            res = os.system(f"python3 {test_case_name_backup} >> {log_path}")
```
Comment (binbinlv) [inline tests/testcases/hub/execute_hub_operators.py:211]: do not rewrite the test case file directly, and generate one file for each model and save it with name including model name to separate them.

File: tests/testcases/hub/execute_hub_operators.py [hunk 3]
```diff
@@ -152,31 +192,42 @@ def execute_ops(op_clone_list):
...
+        if is_succeed == False:
             op_fail_list.append(op_address)
         else:
-            logger.info("Success to run readme for %d: operator %s" % (op_id,op_address))
             success += 1
         op_id += 1
-
     if success != len(op_clone_list):
         logger.error("Fail to run readme for [%d] operators: [%s]" % (len(op_fail_list), op_fail_list))
+        logger.error("Failed models: [%s]" % model_fail_list)
```
Comment (binbinlv) [inline tests/testcases/hub/execute_hub_operators.py:225]: Could you print models  for each op, only models may not reflect what op it owned by.

File: tests/testcases/hub/execute_hub_operators.py (hunk 1 above)
Comment (NicoYuan1986) [reply tests/testcases/hub/execute_hub_operators.py]: > I think here should be modify_specific_model(test_case_name_backup, model_name). And move this line after "test_case_name_backup = test_case_path + "/test_" + op_name_single + ".py""

Done.

File: tests/testcases/hub/execute_hub_operators.py (hunk 3 above)
Comment (NicoYuan1986) [reply tests/testcases/hub/execute_hub_operators.py:243]: > Could you print models for each op, only models may not reflect what op it owned by.

Sentence 'model_fail_list.append(op_address + ': ' + model_name)' has been modified to achieve this step.

File: tests/testcases/hub/execute_hub_operators.py (hunk 2 above)
Comment (NicoYuan1986) [reply tests/testcases/hub/execute_hub_operators.py]: > do not rewrite the test case file directly, and generate one file for each model and save it with name including model name to separate them.

New file has been generated.

File: .github/workflows/weekly_ci2.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    #        │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    - cron: "0 0 * * 6"
+  pull_request:
+    # file paths to consider in the event
+    paths:
+      - '.github/workflows/weekly_ci.yml'
```
Comment (binbinlv) [inline .github/workflows/weekly_ci2.yml:18]: here should be ".github/workflows/weekly_ci2.yml'

File: .github/workflows/weekly_ci2.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    # * is a special character in YAML so you have to quote this string
+    #        ┌───────────── minute (0 - 59)
+    #        │ ┌───────────── hour (0 - 23)
+    #        │ │ ┌───────────── day of the month (1 - 31)
+    #        │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    - cron: "0 0 * * 6"
```
Comment (binbinlv) [inline .github/workflows/weekly_ci2.yml:14]: Please modify to other time separated from weekly ci part 1

File: .github/workflows/weekly_gpu_ci2.yml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    #        │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    - cron: "0 0 * * 0"
+  pull_request:
+    # file paths to consider in the event
+    paths:
+      - '.github/workflows/weekly_gpu_ci.yml'
```
Comment (binbinlv) [inline .github/workflows/weekly_gpu_ci2.yml:18]: Same error.

**Pull request closed without merging**